sp-docs/docs/build.rst

```diff
@@ -0,0 +1,67 @@
+.. _build:
+
+Build and preview
+=================
+
+The starter pack provides make commands to build and view the documentation.
+All these commands will automatically set up the Python environment if it isn't set up yet.
+
+.. _build-docs:
+
+Build the documentation
+-----------------------
+
+To build the documentation, run the following command::
+
+  make html
+
+This command installs the required tools and renders the output to the :file:`_build/` directory.
```

Comment:
Nitpick.

It might be obvious to some but maybe specify where this `_build` directory is? e.g. relative to the repo root or relative to the 'docs' folder?